### ISSUE TITLE
Please provide a build for ubuntu 18.04 mono 7.8.2

### ISSUE THREAD

**Issue by vivekcek** [lextudio/monodevelop.netcoredbg#3]
I tried to build this with cmake and core but did not got any bin created. So could you please build and provide me binaries of this branch.

https://github.com/lextm/netcoredbg/tree/test

**Comment by lextm** [comment]
https://github.com/lextudio/monodevelop.netcoredbg/releases

**Comment by vivekcek** [comment]
I added this to mono still no break point is hitting. Do i need to add this to path variable?

**Comment by vivekcek** [comment]
THis my path to debugger configured "/home/vivek/debugnet/bin/netcoredbg"
And /home/vivek/debugnet/bin/netcoredbg --help will work (from terminal)

**Comment by lextm** [comment]
What kind of project are you testing? `dotnet new console` should be a good test case to run before moving to a more complex project.

**Comment by vivekcek** [comment]
Yes break point working for console but not for a web api project. What to do please help. Target framework is 2.2

**Comment by lextm** [comment]
It seems to be something Linux only.

**Comment by vivekcek** [comment]
Yes I am using this with Ubuntu 18.04 mono 7.8. Is there any way this to work for web api project

**Comment by vivekcek** [comment]
lex any help for web api projects in Ubuntu mono

**Comment by vivekcek** [comment]
Any update?

**Comment by lextm** [comment]
No plan to work on it. PR is welcome.

**Comment by vivekcek** [comment]
> No plan to work on it. PR is welcome.

Is the issue is in extension or necoredbg library?
